core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -671,7 +675,10 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
       classOf[LongWritable],
       classOf[BytesWritable],
       conf=conf)
-    val data = br.map{ case (k, v) => v.getBytes}
+    val data = br.map{ case (k, v) =>
```

Comment:
Super-minor style nit, but I think our house-style here is to have a space before the opening `{` and to place the closing `}` on its own line.